.github/workflows/javadocs.yml

```diff
@@ -47,8 +47,7 @@ jobs:
             cd fhir
             export JAVA_HOME=/opt/hostedtoolcache/AdoptOpenJDK/1.0.0-ga-11-jdk-hotspot-linux-x64-normal-latest/x64/
             mvn -ntp -B -N -Pset-version-release clean -f fhir-parent -Ddeploy.version=${TAG_VERSION} -Dfhir-tools.version=${TAG_VERSION} -Dmaven.wagon.http.retryHandler.count=3
-            mvn -ntp -B install --file fhir-parent -DskipTests -Ddeploy.version=${TAG_VERSION} -Dfhir-tools.version=${TAG_VERSION} -Dmaven.wagon.http.retryHandler.count=3
-            mvn -ntp -B site --file fhir-parent -DskipTests -Ddeploy.version=${TAG_VERSION} -Dfhir-tools.version=${TAG_VERSION} -Dmaven.wagon.http.retryHandler.count=3
+            mvn -ntp -DskipTests -B clean install org.apache.maven.plugins:maven-javadoc-plugin:3.0.1:javadoc-no-fork org.apache.maven.plugins:maven-javadoc-plugin:3.0.1:test-javadoc-no-fork org.apache.maven.plugins:maven-javadoc-plugin:3.0.1:aggregate -f fhir-parent  -Ddeploy.version=${TAG_VERSION} -Dfhir-tools.version=${TAG_VERSION}  -Dmaven.wagon.http.retryHandler.count=3
```

Comment:
so instead of using the mvn site plugin, this is using `javadoc-no-fork`, `test-javadoc-no-fork`, and `aggregate`?  do i have that right?

Reply:
that's right, and it appears to work properly locally